src/components/divider/divider.ts

```diff
@@ -0,0 +1,70 @@
+import { LitElement, html } from 'lit';
+import { property } from 'lit/decorators.js';
+import { themes } from '../../theming/theming-decorator.js';
+import { registerComponent } from '../common/definitions/register.js';
+import { styles } from './themes/divider.base.css.js';
+import { styles as shared } from './themes/shared/divider.common.css.js';
+import { all } from './themes/themes.js';
+
+/**
+ * The igc-divider allows the content author to easily create a horizontal/vertical rule as a break between content to better organize information on a page.
+ *
+ * @element igc-divider
+ *
+ */
+
+/**
+ * Enum for the type, determining if the divider is solid or dashed.
+ */
```

Comment:
I suppose this is a left over from some enumeration type.
Anyway, you should remove it as this now acts as the JSDoc for the class.